ci/test_python_other.sh

```diff
@@ -37,7 +38,7 @@ rapids-logger "pytest dask_cudf (legacy)"
 DASK_DATAFRAME__QUERY_PLANNING=False ./ci/run_dask_cudf_pytests.sh \
   --junitxml="${RAPIDS_TESTS_DIR}/junit-dask-cudf-legacy.xml" \
   --numprocesses=8 \
-  --dist=loadscope \
+  --dist=worksteal \
```

Comment:
I think this line is just an unrelated update?

Reply:
Yes, I'm normalizing these. The dask-cudf "legacy" tests were added around the same time that we changed this strategy to `worksteal`, if I recall correctly. This is probably an oversight, which we missed due to merge order.